src/app/shared/components/button/Button.tsx

```diff
@@ -3,7 +3,7 @@ import './button.scss';
 
 class Button extends Component {
```

Comment:
This component can be a functional component and pass all the props to maintain the button functionalities

Reply:
You are right.